Ginger/GingerCore/Drivers/ASCF/ASCFDriver.cs

```diff
@@ -1481,6 +1481,11 @@ public bool IsLiveSpySupported()
             return false;
         }
 
+        public bool IsWinowSelectionRequired()
```

Comment:
I don't think this supposed to be part of the Driver interface but part of the PlatformInfo.
Driver interface should include operations which Driver needs to do, like GetElementInfo(), but things like, titles to use, or if to show this configuration or another needs to be on PlatformInfo unless very specific to Driver type inside same Platform. 
Need to change in next PR